cdap-docs/integrations/source/partners/cloudera/configuring.rst

```diff
@@ -186,16 +188,82 @@ The following is the generic procedure for Major/Minor version upgrades:
 
 .. _cloudera-release-specific-upgrade-notes:
 
-.. rubric:: Upgrading CDAP 3.0 to 3.1
+.. rubric:: Upgrading CDAP 3.0 to 3.1 and Upgrading CDH 5.3 to 5.4
+
+**Background:** CDH 5.3 ships with HBase 0.98 while CDH 5.4 ships with HBase 1.0. We support
+CDH 5.4 as of CDAP 3.1.0. Upgrading from CDH 5.3 to CDH 5.4 includes an HBase upgrade in
+addition to a CDAP upgrade. 
+
+**It is important to perform these steps as described, otherwise you can end up with an
+unusable system.** The coprocessors may not get upgraded correctly, and HBase
+regionservers may crash.
+
+If you make a mistake and get stuck, see these troubleshooting instructions for
```

Comment:
Instead of `If you make a mistake and get stuck`, can we say `In case something goes wrong with upgrade, and you end up with a dead HBase Region Server, see these troubleshooting instructions for...`